src/app/modules/learning/knowledge-component/submission-result/submission-result.component.scss

```diff
@@ -39,4 +39,10 @@
 
 .feedback-container > pre:last-child {
   scroll-snap-align: end;
-}
\ No newline at end of file
+}
+
+#ie-button, #ae-button {
```

Comment:
Do we need this?

Reply:
Turns out we don't. Removed id from html also.